archive-bibliotech.php

```diff
@@ -14,7 +14,7 @@
  *
  * @package WordPress
  * @subpackage Twenty_Twelve
- * @since Twenty Twelve 1.0
+ * @since 1.0
```

Comment:
I don't think we actually need archive-bibliotech.php file, I don't think
it is being used... that's why I didn't alter the previous comments block.

2016-07-15 10:29 GMT-04:00 Matthew Bernhardt <notifications@github.com>:

> In archive-bibliotech.php
> <https://github.com/MITLibraries/MITLibraries-news/pull/69#discussion_r70982094>
> :
>
> > @@ -14,7 +14,7 @@
> >   *
> >   * @package WordPress
> >   * @subpackage Twenty_Twelve
> > - * @since Twenty Twelve 1.0
> > + * @since 1.0
>
> This template should not be ascribed to WordPress, or Twenty_Twelve - it
> was created specifically for this theme.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MITLibraries/MITLibraries-news/pull/69/files/681ea9151eb773f0fde949e71eccd585b468c1e9#r70982094>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABv1VaLikfOUVzvexoaJCuONxjLwuIDzks5qV5lPgaJpZM4JLyGJ>
> .
>



-- 

"The reason is never the reason."


*- Some French Movie Director*"Every portrait that is painted with feeling
is a portrait of the artist, not the sitter."
*- Oscar Wilde*